api/configuration/v1alpha1/kongservice_types.go

```diff
@@ -88,9 +88,9 @@ func (c *KongService) SetConditions(conditions []metav1.Condition) {
 
 // KongServiceSpec defines specification of a Kong Route.
 type KongServiceSpec struct {
-	// ControlPlaneRef is a reference to a ControlPlane this Route is associated with.
-	// +kubebuilder:validation:Required
-	ControlPlaneRef ControlPlaneRef `json:"controlPlaneRef"`
+	// ControlPlaneRef is a reference to a ControlPlane this KongService is associated with.
+	// +kubebuilder:validation:Optional
```

Comment:
```suggestion
	// +optional
```
To make it coherent with the rest